week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -7,18 +7,24 @@
   NOTE: only the names should be returned, not the means of transport.
 */
 
-function journeyPlanner() {
-
+function journeyPlanner(select, vehicle) {
```

Comment:
So this is correct but it could be made even better by using the filter function that we learnt in class

```
select.filter(select=>select.includes(vehicle)).map(select[0])
```